### DISCUSSION TITLE
蘋果接受台積電漲價決定，影響輝達也同意漲價

### DISCUSSION THREAD

**Discussion by dinosaur1009** [1101-GS4719-C/Python-Community-GS4719-C#744] [科技新知]
https://finance.technews.tw/2022/10/04/apple-accepts-tsmcs-price-hike-decision/
外媒 Tom′s Hardware 報導，市場消息表示拒絕接受台積電 2023 年漲價的蘋果，現在傳出接受漲價的消息。而蘋果一貫對非官方消息拒絕評論。
日前台灣媒體報導，部分 IC 設計客戶接到台積電 2023 年漲價通知，但台積電前五大客戶的蘋果與輝達都拒絕接受。

來源指台積電 2023 年將 8 吋晶圓價格調高 6%，12 吋晶圓價格調高 3%~5%。蘋果是台積電最大的客戶，據說蘋果訂單占整體營收 25%。然蘋果也不能動搖台積電漲價決定，蘋果還是接受台積電漲價。這可能使同樣拒絕漲價的輝達只能同意，因單一客戶不大可能使台積電撤回漲價決定。

台積電能順利漲價在擁有廣泛客群。晶圓代工雖於整體大環境不佳，營收受嚴重衝擊，尤其依賴消費性產品如智慧手機、顯卡、儲存設備的公司衝擊最大。但反觀台積電下半年客戶需求依舊強勁，因台積電業務成長動力來自汽車電子、資料中心及高效能運算，與其他晶圓代工廠商有很大不同。

之前市場繁榮且晶片荒持續許久，相較其他晶圓代工廠紛紛漲價，台積電僅微調價格，接下來台積電漲價市場認為合理，可抵消全球通膨壓力下原物料成本上漲。

感覺台積電真的還是很強力的在主導市場呢.....不愧是台積電

**Comment by Yunhans** [comment]
台積電漲價，蘋果會不會也一起漲價

**Comment by manipulateed** [comment]
牽一髮而動全球，台積電的影響力不容小覷。

**Comment by williamyen0623** [comment]
all in 台積電 護國人 我的超人

**Comment by Pietra1226** [comment]
全球大漲價，看來又有一段時間不能組電腦了

**Comment by danwan1027** [comment]
東西又要漲價了

**Comment by CPCpyt** [comment]
供應鏈漲價帶動產品家族的大漲價，
很期待在日後台積電的擴展能返創造更新穎的價值。

**Comment by ruthchen1020** [comment]
台積電還是對世界有著極大的以響力，漲價也可以審視客戶的品牌個性。
